source/main.cpp

```diff
@@ -382,13 +382,19 @@ void GameLoop(PlayerInfo &player, TaskQueue &queue, const Conversation &conversa
 			}
 			else
 			{
-				if(frameRate < 60)
+				int targetFrameRate = !isFastForward && Preferences::Has("Half speed") ? 30 : 60;
```

Comment:
isFastForward should be renamed to something like speedToggle

Reply:
I wonder if we could introduce an enum for that purpose.

Reply:
> isFastForward should be renamed to something like speedToggle

It's fast forward relative to the game speed, whether that's normal or halved, so I think it's OK?